pkg/operatorsource/configuring.go

```diff
@@ -102,6 +102,12 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *v1alpha1.Oper
 		WithOwnerLabel(in).
 		CatalogSourceConfig()
 
+	// Drop the status to force a CatalogSourceConfig update. This is to account
+	// for the the scenario where a Quay namespace has changed without
+	// app-registry repositories being added or removed but with existing
+	// repositories being updated.
+	cscUpdate.Status = v1alpha1.CatalogSourceConfigStatus{}
```

Comment:
If we are going to do this, why not just remove the invalid cache check?

Just for a short term fix?

Reply:
Do you mean to remove the `IsEntryStale()` check? Won't that cause the `CatalogSourceConfig` to get updated on every kube resync interval?

Reply:
No, I mean that we should do something to the csc that makes `IsEntryStale` return true in this case.

This is a nit either way..
 Let's figure it out later to get something working for now.